tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -623,11 +623,8 @@ const assertBreadcrumbIsDisplayedFor = async function(resource, clickable, nonCl
   let isBreadcrumbVisible = false
 
   // Check if the breadcrumb is visible
-  await client.element('xpath', resourceBreadcrumbXpath, result => {
-    if (result.status > -1) {
-      isBreadcrumbVisible = true
-    }
-  })
+  await client.useXpath().waitForElementVisible(resourceBreadcrumbXpath)
+  isBreadcrumbVisible = true
```

Comment:
I don't think the code will ever come here, because `waitForElementVisible` will fail and exit the test?

Reply:
fixed